src/filters/multipart.rs

```diff
@@ -61,7 +61,13 @@ impl FormOptions {
     ///
     /// Defaults to 2MB.
     pub fn max_length(mut self, max: u64) -> Self {
-        self.max_length = max;
+        self.max_length = Some(max);
+        self
+    }
+
+    /// Set no limit to maximum byte length for the body.
+    pub fn no_max_length(mut self) -> Self {
```

Comment:
We could change to `max_length(impl Into<Option<u64>>)`. All previous calls will still work, but you can now pass `None`. And then document that.

Reply:
Sounds good, thanks!

Reply:
Done.